pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BrokerUserDefinedMessageHandlerFactory.java

```diff
@@ -136,8 +140,7 @@ public HelixTaskResult handleMessage() {
     }
 
     @Override
-    public void onError(Exception e, ErrorCode errorCode, ErrorType errorType) {
-      LOGGER.error("Caught exception on table: {} (code: {}, type: {})", _tableNameWithType, errorCode, errorType, e);
+    public void onError(Exception e, ErrorCode code, ErrorType type) {
```

Comment:
It is better to leave the error message here, in case there is some issue with Helix. Most likely it wont be called anyway.

Reply:
Done